src/main/java/pw/itr0/kaba/encrypt/PasswordBasedDecrypter.java

```diff
@@ -0,0 +1,38 @@
+package pw.itr0.kaba.encrypt;
+
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+/**
+ * Interface for class to decrypt byte array based on password.
+ * <p>
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [<p> tag should be placed immediately before the first word, with no space after.](https://www.codacy.com/app/ryotan/kaba-vault/file/2578430307/issues/source?bid=3298589&fileBranchId=3301306#l8)